modules/building-memcached-operator-using-osdk.adoc

```diff
@@ -69,6 +70,60 @@ update the generated code for that resource type:
 $ operator-sdk generate k8s
 ----
 
+. *Optional: Add custom validation to your CRD.*
++
+OpenAPIv3 schemas are added to CRD manifests in the `spec.validation` block when
+the manifests are generated. This validation block allows Kubernetes to validate
+the properties in a Memcached CR when it is created or updated.
++
+Additionally, a `pkg/apis/<group>/<version>/zz_generated.openapi.go` file is
```

Comment:
This paragraph can be removed, since we've deprecated the command generating this file.

Reply:
Since the 4.3 release uses v0.12.0, this section should be kept as-is.